src/sagas/wallet.js

```diff
@@ -247,16 +247,16 @@ export function* startWallet(action) {
     console.log('[+] Start wallet.', serverInfo);
 
     let version;
-    let networkName = network.name;
+    let serverNetworkName = networkName;
 
     if (serverInfo) {
       version = serverInfo.version;
-      networkName = serverInfo.network && serverInfo.network.split('-')[0];
+      serverNetworkName = serverInfo.network && serverInfo.network.split('-')[0];
     }
 
     yield put(setServerInfo({
```

Comment:
Why is this information used? The `networkName` from the lib might be different from the server

Reply:
This information is later used by the `fetchTokenMetadata()` saga:
https://github.com/HathorNetwork/hathor-wallet/blob/9a477a368bc10415325b29d2e52dc9ed566731fc/src/sagas/tokens.js#L161-L167

The network name obtained directly from the server may have a suffix, and that's why we treat it with a `.split('-')[0]`, but it should not be different from the network name.

Do you see any cases where the names may not match and cause issues?

Reply:
Yes, the nano network in the full node is `nano-testnet-alpha`. This will add conflict to my wallet desktop code for nano but I can handle it there, no problem.